xen/arch/arm/tee/ffa_partinfo.c

```diff
@@ -61,12 +61,15 @@ static int32_t ffa_partition_info_get(uint32_t w1, uint32_t w2, uint32_t w3,
 }
 
 int32_t ffa_handle_partition_info_get(uint32_t w1, uint32_t w2, uint32_t w3,
-                                      uint32_t w4, uint32_t w5, uint32_t *count,
-                                      uint32_t *fpi_size)
+                                      uint32_t w4, uint32_t w5,
+                                      uint32_t *count, uint32_t *size)
 {
-    int32_t ret = FFA_RET_DENIED;
+    int32_t ret;
     struct domain *d = current->domain;
     struct ffa_ctx *ctx = d->arch.tee;
+    uint32_t src_size, dst_size;
+    void *dst_buf;
```

Comment:
Since we're doing arithmetic on this pointer, should it be an `uint8_t *`?

Reply:
arithmetic on void pointers is allowed in xen